jakartaee9/JakartaEE9ReleasePlan.md

```diff
@@ -57,7 +57,7 @@ Jakarta EE 9 WILL NOT impose any backward compatibility requirements for compati
 
 ### Java SE Version
 
-For inclusion in Jakarta EE 9, specification’s APIs MUST be compiled at the Java SE 8 source level.  However, compatible implementations of the Jakarta EE 9 Web Profile and Full Profile MUST certify compatibility on Java SE 11. Compatible Implementations MAY additionally certify and support Java SE 8.
+For inclusion in Jakarta EE 9, specification’s APIs MUST be compiled at the Java SE 8 source level.  Compatible Implementations of the Jakarta EE 9 Platform and Web Profile MUST certify compatibility on Java SE 8. Compatible Implementations MAY additionally certify and support Java SE 11.
```

Comment:
>  Compatible Implementations MAY additionally certify and support Java SE 11.

Is this sentence still necessary?  If so, maybe it could be rephrased as:
 Compatible Implementations MAY additionally certify and support Java SE 11 or above.

For example, are we going to complain if somebody claims compatibility with Java 8 and Java 14, but not Java 11?

Reply:
```
Is this sentence still necessary? If so, maybe it could be rephrased as:
Compatible Implementations MAY additionally certify and support Java SE 11 or above.
```
The Jakarta EE 9 TCK tests should only be run with (required) Java SE 8 and optionally Java SE 11.  

Is the "support" word implying that Java SE 14 couldn't be supported by a Jakarta EE 9 server?

Would stating without "support" be clearer? Perhaps like:
```
Compatible Implementations of the Jakarta EE 9 Platform and Web Profile MAY certify compatibility on Java SE 11.
```
OR
```
Compatible Implementations of the Jakarta EE 9 Platform and Web Profile MAY additionally certify compatibility on Java SE 11.
```

Reply:
> > Compatible Implementations MAY additionally certify and support Java SE 11.
> 
> Is this sentence still necessary? If so, maybe it could be rephrased as:
> Compatible Implementations MAY additionally certify and support Java SE 11 or above.
> 
> For example, are we going to complain if somebody claims compatibility with Java 8 and Java 14, but not Java 11?

Good catch, Andy.  My suggested wording to the Steering Committee was this, but I didn't update the PR appropriately...

"Compatible Implementations MAY additionally certify and support later versions of the Java SE runtime."